scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -696,6 +791,87 @@ cdef _dijkstra_undirected(
     free(nodes)
 
 
+cdef _dijkstra_undirected_multi(
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] source_indices,
+            np.ndarray[DTYPE_t, ndim=1, mode='c'] csr_weights,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] csr_indices,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] csr_indptr,
+            np.ndarray[DTYPE_t, ndim=1, mode='c'] csrT_weights,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] csrT_indices,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] csrT_indptr,
+            np.ndarray[DTYPE_t, ndim=1, mode='c'] dist_matrix,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] pred,
```

Comment:
probably want typed memoryviews as noted above

Reply:
this is now addressed I believed.